src/cladetime/cladetime.py

```diff
@@ -15,51 +15,45 @@
 
 
 class CladeTime:
-    """
-    Wrapper around Nextstrain/Nextclade tooling to generate Sars-CoV-2 genome clade assignments
-    and aggregations at a specific point in time. CladeTime operates on Genbank sequences.
+    """Interface for Nextstrain Sars-CoV-2 genome sequences and clades.
+
+    The CladeTime class is instantiated with two optional arguments that
+    specify the point in time at which to access genome sequences/metadata
+    as well as the reference tree used for clade assignment. CladeTime
+    interacts with GenBank-based data provided by the Nextstrain project.
+
+    Parameters
+    ----------
+    sequence_as_of : datetime.datetime | str | None
+        Sets the versions of Nextstrain SAR-CoV-2 genome sequence
+        and sequence metadata files that will be used by
+        CladeTime properties and methods. Can be a datetime object or a
+        string in YYYY-MM-DD format, both of which will be treated as
+        UTC. The default value is midnight of the current UTC date.
```

Comment:
```suggestion
        UTC. The default value is the current time.
```

Reply:
(based on https://github.com/reichlab/cladetime/pull/35/files#diff-4dc90011ce53fb53bb9eac472b0a5672c1375e8bdcf6f8e7283bd9fdea17b746L171)

Reply:
Thanks! Will take a closer look at this--IIRC, if you pass in YYYY-MM-DD or a datetime object w/o a time, it will default to midnight, but I might be misrembering. Either way, makes sense to clarify.